aws/ecs/packer/to-upload/peers.sh

```diff
@@ -2,29 +2,221 @@
 # This script print a list of IPs of the EC2 instances in your current Autoscaling Group.
 set -eu
 
-aws=/usr/local/bin/aws
+__instance_tag_key="${1:-WEAVE_ECS}"
```

Comment:
Why the `__` prefixes? Can you try to make the style (uppercase variables `function` prefixes) more consistent with existing code like https://github.com/weaveworks/integrations/blob/master/aws/ecs/packer/to-upload/run.sh ? I know we where already using lowercase and uppercase variables here but let's go for the latter.